app/Console/ReplayCommand.php

```diff
@@ -76,7 +76,7 @@ protected function configure()
         );
 
         $this
-            ->setName('replay')
+            ->setName('offer:replay')
```

Comment:
Isn't it also possible to replay organizers, roles, labels etc through this? Looking at the `AggregateType` options used just above this, there are more options than just offers (events/places):
```
class AggregateType extends Enum
{
    const EVENT = 'event';
    const PLACE = 'place';
    const VARIATION = 'variation';
    const ORGANIZER = 'organizer';
    const MEDIA_OBJECT = 'media_object';
    const ROLE = 'role';
    const LABEL = 'label';
}
```

Reply:
I was a bit too drastic with this so I reverted it back to the default namespace.